File: sample/src/main/java/com/github/lany192/box/sample/activity/AboutActivity.java
package com.github.lany192.box.sample.activity;

import android.os.Bundle;

import com.github.lany192.box.activity.ViewBindingActivity;
import com.github.lany192.box.sample.databinding.ActivityAboutBinding;
import com.github.lany192.box.view.ActivityBinding;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

/**
 * @author Administrator
 */
public class AboutActivity extends ViewBindingActivity<ActivityAboutBinding> {

    @Override
    public void init(@Nullable Bundle savedInstanceState) {
//        getBinding().getContent().ge
    }

    @NotNull
    @Override
    public ActivityAboutBinding getContentView() {
        return ActivityAboutBinding.inflate(getLayoutInflater());
    }
}

File: sample/src/main/java/com/github/lany192/box/sample/activity/AboutActivity.kt
package com.github.lany192.box.sample.activity

import android.os.Bundle
import com.github.lany192.box.activity.ActivityConfig
import com.github.lany192.box.activity.BaseActivity
import com.github.lany192.box.sample.R

class AboutActivity : BaseActivity() {

    override fun getConfig(): ActivityConfig {
        return ActivityConfig().apply {
            layoutId = R.layout.activity_about
            statusBarDarkFont = true
            transparentStatusBar = true
            toolBarLayoutId = R.layout.toolbar_default
            keyboardEnable = true
            hasBackBtn = true
        }
    }

    override fun init(savedInstanceState: Bundle) {}
}